src/dotnet/Fable.Tools/ProjectCracker.fs

```diff
@@ -242,8 +252,9 @@ let getPaketProjRefs paketDir projFile =
             []
 
 let getProjectOptionsFromFsproj projFile =
-    let paketDir = tryFindPaketDirFromProject projFile
-    paketDir |> Option.iter checkFableCoreVersion
+    //let paketDir = tryFindPaketDirFromProject projFile
+    //paketDir |> Option.iter checkFableCoreVersion
+    let paketDir = None
```

Comment:
Actually if I put this back, it causes the errors I am getting, I will put these lines back:
```fs
    let paketDir = tryFindPaketDirFromProject projFile
    paketDir |> Option.iter checkFableCoreVersion
```
and have `tryFindPaketDirFromProject` actually return None when an exception is thrown, i.e.:
```fs
let tryFindPaketDirFromProject projFile =
    let projDir = Path.GetDirectoryName(projFile)
    if File.Exists(Path.Combine(projDir, "paket.references"))
    then 
        try 
            findPaketDependenciesDir projDir [] |> Some
        with
        | _ -> None
    else None
```